docker.hcl

```diff
@@ -0,0 +1,246 @@
+    image_tag("server", BRANCH_NAME)
+  ]
+}
+
+target "storage" {
+  inherits = ["service-base", get_target()]
+  context = "${PWD}/packages/services/storage/dist"
+  args = {
+    IMAGE_TITLE = "graphql-hive/storage"
+    IMAGE_DESCRIPTION = "The storage service of the GraphQL Hive project."
+  }
+  tags = [
+    image_tag("storage", COMMIT_SHA),
+    image_tag("storage", BRANCH_NAME)
+  ]
+}
+
+target "stripe-billing" {
+  inherits = ["service-base", get_target()]
+  context = "${PWD}/packages/services/stripe-billing/dist"
+  args = {
+    IMAGE_TITLE = "graphql-hive/stripe-billing"
+    IMAGE_DESCRIPTION = "The stripe billing service of the GraphQL Hive project."
+    HEALTHCHECK_CMD = "wget --spider -q http://127.0.0.1:3010/_readiness"
+  }
+  tags = [
+    image_tag("stripe-billing", COMMIT_SHA),
+    image_tag("stripe-billing", BRANCH_NAME)
+  ]
+}
+
+target "tokens" {
+  inherits = ["service-base", get_target()]
+  context = "${PWD}/packages/services/tokens/dist"
+  args = {
+    IMAGE_TITLE = "graphql-hive/tokens"
+    IMAGE_DESCRIPTION = "The tokens service of the GraphQL Hive project."
+    HEALTHCHECK_CMD = "wget --spider -q http://127.0.0.1:3003/_readiness"
+  }
+  tags = [
+    image_tag("tokens", COMMIT_SHA),
+    image_tag("tokens", BRANCH_NAME)
+  ]
+}
+
+target "usage-estimator" {
+  inherits = ["service-base", get_target()]
+  context = "${PWD}/packages/services/usage-estimator/dist"
+  args = {
+    IMAGE_TITLE = "graphql-hive/usage-estimator"
+    IMAGE_DESCRIPTION = "The usage estimator service of the GraphQL Hive project."
+    HEALTHCHECK_CMD = "wget --spider -q http://127.0.0.1:3008/_readiness"
+  }
+  tags = [
+    image_tag("usage-estimator", COMMIT_SHA),
+    image_tag("usage-estimator", BRANCH_NAME)
+  ]
+}
+
+target "usage-ingestor" {
+  inherits = ["service-base", get_target()]
+  context = "${PWD}/packages/services/usage-ingestor/dist"
+  args = {
+    IMAGE_TITLE = "graphql-hive/usage-ingestor"
+    IMAGE_DESCRIPTION = "The usage ingestor service of the GraphQL Hive project."
+    HEALTHCHECK_CMD = "wget --spider -q http://127.0.0.1:3007/_readiness"
+  }
+  tags = [
+    image_tag("usage-ingestor", COMMIT_SHA),
+    image_tag("usage-ingestor", BRANCH_NAME)
+  ]
+}
+
+target "usage" {
+  inherits = ["service-base", get_target()]
+  context = "${PWD}/packages/services/usage/dist"
+  args = {
+    IMAGE_TITLE = "graphql-hive/usage"
+    IMAGE_DESCRIPTION = "The usage ingestor service of the GraphQL Hive project."
+    HEALTHCHECK_CMD = "wget --spider -q http://127.0.0.1:3006/_readiness"
+  }
+  tags = [
+    image_tag("usage", COMMIT_SHA),
+    image_tag("usage", BRANCH_NAME)
+  ]
+}
+
+target "webhooks" {
+  inherits = ["service-base", get_target()]
+  context = "${PWD}/packages/services/webhooks/dist"
+  args = {
+    IMAGE_TITLE = "graphql-hive/webhooks"
+    IMAGE_DESCRIPTION = "The webhooks ingestor service of the GraphQL Hive project."
+    HEALTHCHECK_CMD = "wget --spider -q http://127.0.0.1:3005/_readiness"
```

Comment:
@n1ru4l Most of the containers accepts the `PORT` env var. Maybe we should inline the healthcheck with the `PORT` variable?

Reply:
storage does not have one, so it would already fail there

Reply:
instead we would have to declare an additional dockerfile

Reply:
But i agree that we should favor the local PORT env variable

Reply:
done in https://github.com/kamilkisiela/graphql-hive/pull/386/commits/2819a6542badf97676e644021c3b641bc13809ba